docs/Developer-Guide.md

```diff
@@ -269,6 +279,18 @@ $ go get -d -u github.com/kata-containers/kata-containers
 $ cd $GOPATH/src/github.com/kata-containers/kata-containers/src/agent && make
 ```
 
+The agent is built with seccomp capability by default.
+If you want to build the agent without the seccomp capability, you need to run `make` with `SECCOMP=no` as follows.
+
+```
+$ cd $GOPATH/src/github.com/kata-containers/kata-containers/src/agent && make SECCOMP=no
```

Comment:
No problem with this, but an alternative is the following (which is slightly shorter ;):

```bash
$ make -C $GOPATH/src/github.com/kata-containers/kata-containers/src/agent SECCOMP=no
```

Reply:
I fixed it.